sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/jsonwrapper/JsonWrapper.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.core.implementation.serializer.jsonwrapper;
+
+import com.azure.core.implementation.serializer.jsonwrapper.api.Config;
+import com.azure.core.implementation.serializer.jsonwrapper.api.JsonApi;
+import com.azure.core.implementation.serializer.jsonwrapper.spi.JsonPlugin;
+
+import java.util.Iterator;
+import java.util.ServiceLoader;
+
+public class JsonWrapper {
+
+    private static final ServiceLoader<JsonPlugin> PLUGIN_LOADER = ServiceLoader.load(JsonPlugin.class);
+
+    /**
+     * Create new instance of JsonApi
+     *
+     * @return JsonApi
+     */
+    public static JsonApi newInstance() {
```

Comment:
There are unchecked conversion warning which cause build failures - given this module is in implementation package and experimental, it should be ok to suppress it for now by applying `@SuppressWarnings("unchecked")`. @JonathanGiles thoughts?

Reply:
Checked with Jonathan offline, @navalev please suppress the warnings for now.